app/platform/__init__.py

```diff
@@ -137,3 +147,232 @@ def edit_collection(grant_id: UUID4, collection_id: UUID4) -> ResponseReturnValu
     return render_template(
         "platform/developers/edit_collection.html", grant=collection.grant, collection=collection, form=form
     )
+
+
+@platform_blueprint.route(
+    "/grants/<uuid:grant_id>/developers/collections/<uuid:collection_id>/sections/add", methods=["GET", "POST"]
+)
+@mhclg_login_required
+@auto_commit_after_request
+def add_section(grant_id: UUID4, collection_id: UUID4) -> ResponseReturnValue:
+    collection = get_collection_schema(collection_id)
+    form = SectionForm(collection_id=collection_id)
+    if form.validate_on_submit():
+        try:
+            assert form.title.data is not None
+            create_section(
+                title=form.title.data,
+                collection_schema=collection,
+            )
+            return redirect(url_for("platform.list_sections", grant_id=grant_id, collection_id=collection_id))
+        except DuplicateValueError as e:
+            field_with_error: Field = getattr(form, e.field_name)
+            field_with_error.errors.append(f"{field_with_error.label.text} already in use")  # type:ignore[attr-defined]
+    return render_template(
+        "platform/developers/add_section.html", grant=collection.grant, collection=collection, form=form
+    )
+
+
+@platform_blueprint.route(
+    "/grants/<uuid:grant_id>/developers/collections/<uuid:collection_id>/sections/", methods=["GET", "POST"]
+)
+@mhclg_login_required
+@auto_commit_after_request
+def list_sections(
+    grant_id: UUID4,
+    collection_id: UUID4,
+) -> ResponseReturnValue:
+    collection_schema = get_collection_schema(collection_id)
+    return render_template(
+        "platform/developers/list_sections.html", grant=collection_schema.grant, collection=collection_schema
+    )
+
+
+@platform_blueprint.route(
+    "/grants/<uuid:grant_id>/developers/collections/<uuid:collection_id>/sections/<uuid:section_id>/move/<string:direction>",
+    methods=["GET"],
```

Comment:
nit: GET does make this a lot easier but it would be 'more proper' as a POST request I suppose 👀

Reply:
Done